assets/js/search.js

```diff
@@ -0,0 +1,124 @@
+(function() {
+    'use stict';
+    var activityCards, find, mapCardToClassName, mapCardToHeading, hideCard,
```

Comment:
My first programming language was C. Some things stick …

It has the benefit that I don't have to mind about declaring a variable later on. They are all declared in one place.

Shall I rather write them like [in the leaderboard.js](https://github.com/mozilla/activate.mozilla.community/blob/55760c993d3697885358f6283e8babcb34dc0928/assets/js/leaderboard.js#L4)?